ports/stm32/boards/stm32g4xx_hal_conf_base.h

```diff
@@ -0,0 +1,325 @@
+/**
+  ******************************************************************************
+  * @file    stm32g4xx_hal_conf.h
+  * @author  MCD Application Team
+  * @brief   HAL configuration template file.
+  *          This file should be copied to the application folder and renamed
+  *          to stm32g4xx_hal_conf.h.
+  ******************************************************************************
+  * @attention
+  *
+  * <h2><center>&copy; Copyright (c) 2019 STMicroelectronics.
+  * All rights reserved.</center></h2>
+  *
+  * This software component is licensed by ST under BSD 3-Clause license,
```

Comment:
As above, to not over complicate licensing, please redo this file based on, eg, `boards/stm32h7xx_hal_conf_base.h`

Reply:
Done.